resolwe/flow/models/secret.py

```diff
@@ -0,0 +1,57 @@
+"""Resolwe secrets model."""
+import uuid
+
+from fernet_fields import EncryptedTextField
+
+from django.conf import settings
+from django.db import models
+
+
+class SecretManager(models.Manager):
+    """Manager for Secret objects."""
+
+    def create_secret(self, value, contributor):
+        """Create a new secret, returning its handle.
+
+        :param value: Secret value to store
+        :param contributor: User owning the secret
+        :return: Secret handle
+        """
+        secret = self.create(value=value, contributor=contributor)
+        return str(secret.handle)
+
+    def get_secret(self, handle, contributor=None):
```

Comment:
Is there a use-case where contributor is not defined? If not i would prefer to make it mandatory.

Reply:
:+1: